documentation.md

```diff
@@ -0,0 +1,25 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+_**If in doubt consult with your security point of contact**_
+
+## Restricted Items
+1. Secret information, such as passwords, secret access keys, private keys, TOTP QR codes, etc.
+1. Network ranges and IP addresses\
+DWP currently have these classed as _sensitive_
```

Comment:
a) I'm not sure this is true (I think the latest rumour has it that network ranges (CIDR blocks) are OK.  However, I've no idea who the authority is on that policy
b) Can we reword to `DWP currently class these as _sensitive_`, which I think reads better

Reply:
unfortunately, this one is not a rumour